packages/core/src/App/Components/Elements/NotificationMessage/notification-banner.jsx

```diff
@@ -1,40 +1,54 @@
 import PropTypes from 'prop-types';
 import React from 'react';
-import { Button, Icon, StaticUrl, Text } from '@deriv/components';
-import { getUrlBase } from '@deriv/shared';
-import { localize, Localize } from '@deriv/translations';
+import { isMobile } from '@deriv/shared';
+import { Button, Icon, Text } from '@deriv/components';
 
-const NotificationBanner = ({ onClick, header, message, button_text, img_src, img_alt, redirect_link }) => (
+const NotificationBanner = ({ header, message, primary_btn, secondary_btn, img_src, img_alt, onClose }) => (
     <div className='notification-banner'>
         <div className='notification-banner--left'>
-            <Text as='h4' weight='bold' className='notification-banner__title'>
-                <Localize i18n_default_text={header} />
```

Comment:
Question: Don't we need to localize these texts anymore or is it localized somewhere else?

Reply:
The existing code doesn't work due to variable usage within the `<Localize>` component, and to answer your question, we're already passing the *localized* strings here:
https://github.com/binary-com/deriv-app/pull/3044/files#diff-33606cacb4e6c33d9dc94e4be05eebccc1ad26c0556b7357b37d7753f231942cR25-R34

We should always `localize` strings outside of shareable components.

Reply:
Ok, this makes perfect sense 👍🏻 Thank you for your detailed explanation Zaki😃